README.md

```diff
@@ -171,4 +171,6 @@ Rendering distance is not customizable.
 | `-rt <boolean>`        | explicitly enable or disable ray-query                           |
 | `-gi <boolean>`        | explicitly enable or disable ray-traced global illumination      |
 | `-ms <boolean>`        | explicitly enable or disable meshlets                            |
+| `-bl <boolesn>`        | explicitly enable or disable bindless                            |
```

Comment:
`-bl` is not intended to be public - this is testing/troubleshooting only

Reply:
thanks for the reminder, I've repushed to keep the `-fxaa`